app/src/main/java/org/sopt/korailtalk/presentation/home/HomeScreen.kt

```diff
@@ -1,55 +1,146 @@
+    viewModel: HomeViewModel = hiltViewModel(),
+    navigateToReservation: (String, String) -> Unit
 ) {
+    val uiState by viewModel.uiState.collectAsState()
+
+
     Column(
-        horizontalAlignment = Alignment.CenterHorizontally,
-        verticalArrangement = Arrangement.Center,
-        modifier = modifier
-            .background(Color.White)
-            .fillMaxSize()
+        modifier
+            .background(KorailTalkTheme.colors.gray50)
+            .fillMaxSize(),
+        horizontalAlignment = Alignment.Start,
+        verticalArrangement = Arrangement.Top,
     ) {
-        Text(
-            "Home",
-            modifier = Modifier.noRippleClickable(
-                onClick = onReservationClick
-            )
+        KorailTalkBasicTopAppBar(
+            navigationIcon = {
+                Icon(
+                    painter = painterResource(id = R.drawable.img_korail_logo),
+                    contentDescription = null,
+                    tint = KorailTalkTheme.colors.white
+                )
+            },
+            actions = {
+                Icon(
+                    painter = painterResource(id = R.drawable.ic_translate),
+                    contentDescription = null,
+                    tint = KorailTalkTheme.colors.white
+                )
+                Icon(
+                    painter = painterResource(id = R.drawable.ic_hamburger),
+                    contentDescription = null,
+                    tint = KorailTalkTheme.colors.white
+                )
+            }
         )
+        Row(
+            Modifier.padding(start = 24.dp, top = 24.dp, end = 24.dp, bottom = 16.dp)
+        ) {
+            Text(
+                "어디로 갈까요,",
+                style = KorailTalkTheme.typography.headline.head4M18,
+                color = KorailTalkTheme.colors.primary700,
+            )
+            Text(
+                "민주 님?",
+                style = KorailTalkTheme.typography.headline.head3Sb18,
+                color = KorailTalkTheme.colors.primary700,
+            )
+        }
+        Box(
+            Modifier
+                .fillMaxWidth()
+                .padding(horizontal = 16.dp, vertical = 20.dp),
+            contentAlignment = Alignment.Center
+        ) {
+            CheckTrainCard(
+                startStation = uiState.startStation,
+                endStation = uiState.endStation,
+                onSwapClick = { viewModel.swapStations() }, // 뷰모델의 함수 호출
+                onReservationClick = {
+                    navigateToReservation(
+                        uiState.startStation,
+                        uiState.endStation
+                    )
+                }
+            )
+        }
+        Spacer(Modifier.height(16.dp))
+        Box(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(horizontal = 16.dp),
+            contentAlignment = Alignment.Center
+        ) {
+            IconGridScreen(items = items)
+        }
```

Comment:
이거 Box로 한번 더 감싸신 이유가 있으실까요?

현재의 `IconGridScreen`에서 modifier를 뚫어주시면, 아래처럼 바로 사용할 수 있을 것 같아요!
Container 영역까지를 컴포넌트로 묶어주는 편이 더 명확할 것 같아요.

```suggestion
        IconGridScreen(
            items = items,
            modifier = Modifier.padding(horizontal = 16.dp)
        )
```

Reply:
contentAlignment = Alignment.Center가 Box 의 인자라고 해서 중앙에 정렬 시키려구 감쌌습니다! modifier로 뚫어주면 간결해지군요@@!!감사합니당